cypress/integration/pages/articles/testsForAMPOnly.js

```diff
@@ -34,6 +36,23 @@ export const testsThatFollowSmokeTestConfigForAMPOnly = ({
           });
       }
     });
+
+    it('should render a placeholder image within a media block', () => {
+      cy.request(`${config[service].pageTypes.articles.path}.json`).then(
+        ({ body }) => {
+          // `video` blocks can also contain audio, so this test
+          // checks if both media types have a placeholder image.
+          const media = getBlockData('video', body);
+          if (media) {
```

Comment:
I don't mind leaving this conditional for now but I question the value of making tests conditional on the data structure while we are trying to make all our test articles uniform.

Is there a way we can look at the Simorgh data already in the page rather than re-fetching it?

Reply:
As it stands this test breaks without the check, because some of our fixtures (Persian) don't contain AV content. I've improved the current solution by replacing the request in the canonical test by reading the data from `window`.